.jenkins/cscs-ault/env-common.sh

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+export SPACK_ROOT="/apps/ault/SSD/pika/spack"
+export SPACK_USER_CONFIG_PATH="${SPACK_ROOT}/../spack-user-config"
+export SPACK_USER_CACHE_PATH="${SCRATCH}/spack-user-cache-jenkins/pika-cache"
+
+set +x
+source "${SPACK_ROOT}/share/spack/setup-env.sh"
+spack load ccache@4.5.1 %gcc@10.2.0
+set -eux
```

Comment:
Why are the `set`s required?

Reply:
This was temporary, I silenced all this output cause I could find my error faster, I will revert it before it's ready